src/app/components/core/north/north-task-modal/north-task-modal.component.ts

```diff
@@ -125,7 +126,7 @@ export class NorthTaskModalComponent implements OnInit, OnChanges {
           /** request completed */
           this.ngProgress.done();
           this.alertService.success('Schedule updated successfully.');
-          this.notify.emit();
+          // this.notify.emit();
```

Comment:
we want to remove this? what this emit was doing earlier?

Reply:
No, It was a mistake. restored